sqlglot/dialects/presto.py

```diff
@@ -166,7 +167,7 @@ def _unix_to_time_sql(self: Presto.Generator, expression: exp.UnixToTime) -> str
     return f"FROM_UNIXTIME(CAST({timestamp} AS DOUBLE) / POW(10, {scale}))"
 
 
-def _jsonextract_sql(self: Presto.Generator, expression: exp.JSONExtract) -> str:
+def jsonextract_sql(self: Generator, expression: exp.JSONExtract) -> str:
```

Comment:
Should we just move this within presto's `Generator` and then use `super()` in Trino instead of importing it?